lectures/greek_square.md

```diff
@@ -512,58 +638,89 @@ $$
 This can be achieved by setting 
 
 $$
-x_{2,0} = -(V^{2,2})^{-1} V^{2,1} = V_{2,1} V_{1,1}^{-1} x_{1,0}
+x_{2,0} = -(V^{2,2})^{-1} V^{2,1} x_{1,0} = V_{2,1} V_{1,1}^{-1} x_{1,0}
 $$ (eq:deactivate2)
 
+Let's verify {eq}`eq:deactivate1` and {eq}`eq:deactivate2` below
 
-We shall encounter equations very similar to {eq}`eq:deactivate1` and {eq}`eq:deactivate2`
-in  this QuantEcon lecture {doc}`money financed government deficits and inflation <money_inflation>`
-and in many other places in dynamic economic theory.
+To deactivate $\lambda_1$ we use {eq}`eq:deactivate1`
 
-### Implementation
+```{code-cell} ipython3
+xd_1 = np.array((x_0[0], 
+                 V[1,1]/V[0,1] * x_0[0]),
+                dtype=np.float64)
 
-Now we implement the algorithm above.
+# Compute x_{1,0}^*
+np.round(V_inv @ xd_1, 8)
+```
 
-First we write a function that iterates $M$
+We find $x_{1,0}^* = 0$.
 
-```{code-cell} ipython3
-:tags: []
+Now we deactivate $\lambda_2$ using {eq}`eq:deactivate2`
 
-def iterate_M(x_0, M, num_steps):
-    # Eigendecomposition of M
-    Λ, V = np.linalg.eig(M)
-    V_inv = np.linalg.inv(V)
-    
-    print(f"eigenvalue:\n{Λ}")
-    print(f"eigenvector:\n{V}")
-    
-    # Initialize the array to store results
-    x = np.zeros((x_0.shape[0], num_steps))
-    
-    # Perform the iterations
-    for t in range(num_steps):
-        x[:, t] = V @ np.diag(Λ**t) @ V_inv @ x_0
-    
-    return x
+```{code-cell} ipython3
+xd_2 = np.array((x_0[0], 
+                 V[1,0]/V[0,0] * x_0[0]), 
+                 dtype=np.float64)
 
-# Define the state transition matrix M
-M = np.array([[2, -(1 - σ)],
-              [1, 0]])
+# Compute x_{2,0}^*
+np.round(V_inv @ xd_2, 8)
+```
 
-# Initial condition vector x_0
-x_0 = np.array([1, 0])
+We find $x_{2,0}^* = 0$.
 
-# Perform the iteration
-xs = iterate_M(x_0, M, num_steps=100)
+```{code-cell} ipython3
+# Simulate with muted λ1 λ2.
+num_steps = 10
+xs_λ1 = iterate_M(xd_1, M, num_steps)[0]
+xs_λ2 = iterate_M(xd_2, M, num_steps)[0]
+
+# Compute ratios y_t / y_{t-1}
+ratios_λ1 = xs_λ1[1, 1:] / xs_λ1[1, :-1]
+ratios_λ2 = xs_λ2[1, 1:] / xs_λ2[1, :-1] 
 ```
 
-Compare the eigenvector to the roots we obtained above
+The following graph shows the ratios $y_t / y_{t-1}$ for the two cases.
 
-```{code-cell} ipython3
-:tags: []
+We find that the ratios converge to $\lambda_2$ in the first case and $\lambda_1$ in the second case.
 
-roots = solve_λs((1, -2, (1 - σ)))
-print(f"roots: {np.round(roots, 8)}")
+```{code-cell} ipython3
```

Comment:
Is there a reason we're not using our usual OOP plotting style here, as in `fix, ax = ...` followed by `ax.plot` etc.

Reply:
Hi @jstac, I did the first plot first and then the second at request in person so I did not put them into OOP plotting style. 

I will migrate it now!